Frontend/ui-driver/src/Screens/HomeScreen/ComponentConfig.purs

```diff
@@ -176,12 +176,13 @@ accessbilityBannerConfig state =
     config' = config  
       {
         backgroundColor = Color.lightPurple,
-        title = getString LEARN_HOW_YOU_CAN_HELP_RIDERS_REQUIRING_SPECIAL_ASSISTANCE,
+        title = getString LEARN_HOW_YOU_CAN_HELP_CUSTOMERS_REQUIRING_SPECIAL_ASSISTANCE,
         titleColor = Color.purple,
-        actionText = "Learn More",
+        actionText = getString LEARN_MORE,
         actionTextColor = Color.purple,
-        imageUrl = "ny_ic_purple_badge,https://assets.juspay.in/beckn/nammayatri/driver/images/ny_ic_purple_badge.png",
-        isBanner = true
+        imageUrl = "ny_ic_purple_badge,"<> (getAssetStoreLink FunctionCall) <>"ny_ic_purple_badge.png",
+        isBanner = true,
+        stroke = "1,"<>"#339747FF"
```

Comment:
Check this color is already Present or not

Reply:
Updated.